source/backend/api/Areas/CompensationRequisition/Controllers/CompensationRequisitionController.cs

```diff
@@ -204,5 +204,26 @@ public IActionResult GetFileCompensations([FromRoute]FileTypes fileType, [FromRo
 
             return new JsonResult(compensations);
         }
+
+        [HttpGet("{id:long}/financials")]
+        [HasPermission(Permissions.CompensationRequisitionView)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(List<CompensationFinancialModel>), 200)]
+        [SwaggerOperation(Tags = new[] { "compensation-requisition" })]
+        [TypeFilter(typeof(NullJsonResultFilter))]
+        public IActionResult GetCompensationRequisitionFinancials([FromRoute] long id)
+        {
+            _logger.LogInformation(
+                "Request received by Controller: {Controller}, Action: {ControllerAction}, User: {User}, DateTime: {DateTime}",
+                nameof(CompensationRequisitionController),
+                nameof(GetCompensationRequisitionFinancials),
+                User.GetUsername(),
+                DateTime.Now);
+            _logger.LogInformation("Dispatching to service: {Service}", _compensationRequisitionService.GetType());
+
+            var compReqFinancials = _compensationRequisitionService.GetCompensationRequisitionFinancials(id);
+
+            return new JsonResult(_mapper.Map<IEnumerable<CompensationRequisitionModel>>(compReqFinancials));
```

Comment:
could you please check that this is the correct mapping?
I would think that **PimsCompReqFinancial** which is returned by "GetCompensationRequisitionFinancials()" should be matched to the model **CompensationFinancialModel**

Instead I see here that it is being mapped to **CompensationRequisitionModel** - does this make sense here?

Reply:
Also, I was expecting to see the ACQ controller method that returned this info before this PR to be removed since it would be replaced by this new endpoint - right? But I do not see a change here that removes that endpoint from the ACQ controller. 

I'm referring to this: 
![image](https://github.com/user-attachments/assets/ad8d2f1f-415a-4708-9c33-c9238aac357a)

Reply:
removed and updated.